src/Apps/Artist/Routes/CV/CVItem.tsx

```diff
@@ -151,16 +153,23 @@ class CVItem extends Component<CVItemProps, CVItemState> {
                       onClick={() => this.loadMore()}
                       loading={this.state.isLoading ? true : false}
                       width={xs ? "100%" : ""}
```

Comment:
@damassi think this could be `width={[ "100%", "" ]}`? 🤔 

It looks like width is a responsive styled systems value...